src/typings/@babel__eslint-parser/index.d.ts

```diff
@@ -0,0 +1,13 @@
+declare module '@babel/eslint-parser' {
```

Comment:
borrowed this from: https://github.com/SonarSource/SonarJS/blob/master/typings/%40babel__eslint-parser/index.d.ts